### PR TITLE
Add accordion

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
- Create shared Accordion list to be used in multiple locations
- Create shared command component
- Add remaining pages

Next PR: Add missing elements like progress status, errors, validation buttons so I can start wiring endpoints

https://github.com/user-attachments/assets/ca3f3559-1136-484a-8132-defb2ed5e0dc

Screenshot 2024-09-18 at 22 49 51

## Type of change (choose one)
- [ ] Chore (refactors, upgrades, etc.)
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Security fix (non-breaking change that fixes a potential vulnerability)
- [ ] Docs
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
- [ ] I've linked any associated issues to this PR.
- [X] I've tested my own changes.

*[Check our Pull Request Guidelines](https://github.com/inngest/inngest/blob/main/docs/PULL_REQUEST_GUIDELINES.md)*

### CODE REVIEW

File: ui/packages/components/src/CodeBlock/CommandBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  const [dark, setDark] = useState(isDark());
+  const editorRef = useRef<MonacoEditorType>(null);
+  const wrapperRef = useRef<HTMLDivElement>(null);
+
+  const monaco = useMonaco();
+  const activeTabContent = currentTabContent || { content: '', readOnly: true, language: 'json' };
+
+  useEffect(() => {
+    // We don't have a DOM ref until we're rendered, so check for dark theme parent classes then
+    if (wrapperRef.current) {
```
Comment (jacobheric) [inline ui/packages/components/src/CodeBlock/CommandBlock.tsx:33]: just curious, does this cause a light flicker before dark?

File: ui/packages/components/src/CodeBlock/CommandBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    const editor = editorRef.current;
+    if (editor) {
+      const contentHeight = Math.min(1000, editor.getContentHeight());
+      wrapperRef.current!.style.height = `${contentHeight}px`;
+      editor.layout();
+    }
+  };
+
+  return (
+    <>
```
Comment (jacobheric) [inline ui/packages/components/src/CodeBlock/CommandBlock.tsx:66]: These extra fragment wrappers don't seem necessary...more content coming soon?

**Review by jacobheric — APPROVED** [review]
Looks good!

File: ui/packages/components/src/CodeBlock/CommandBlock.tsx (hunk 2 above)
Comment (anafilipadealmeida) [reply ui/packages/components/src/CodeBlock/CommandBlock.tsx:66]: There's one there that can go, was a leftover from a failed experiment with some monaco height stuff

File: ui/packages/components/src/CodeBlock/CommandBlock.tsx (hunk 1 above)
Comment (anafilipadealmeida) [reply ui/packages/components/src/CodeBlock/CommandBlock.tsx:33]: This is the same logic you implemented in the CodeBlock. We need to change that theme logic, as besides the flicker when applying the theme, it also doesn't automatically change it to the other theme when moving between light and dark mode without a render, but I don't wanna mess with it in this PR

**Pull request merged**